docs/tutorials/google/visualizing_calibration_metrics.ipynb

```diff
@@ -33,13 +33,13 @@
   {
    "cell_type": "markdown",
    "metadata": {
-    "id": "8c466cef02c8"
+    "id": "aL6Ny9Y2Rv10"
    },
    "source": [
     "# Calibrations\n",
-    "The [tutorial on calibration](https://quantumai.google/cirq/google/calibration) gives an introduction to calibration\n",
-    "metrics and how to retrieve them from the Google Quantum Computing Service (QCS). In this tutorial, we show to visualize these calibration metrics\n",
-    "via single qubit heatmaps and two qubit interaction heatmaps."
+    "The [tutorial on calibration](https://quantumai.google/cirq/google/calibration) gives an introduction to calibration metrics and how to retrieve them from the Google Quantum Computing Service (QCS). In this tutorial, we show how to visualize these calibration metrics via single qubit heatmaps, two qubit interaction heatmaps and integrated histograms.\n",
+    "\n",
+    "## Disclaimer: The calibration metrics of the quantum processor shown in the tutorial do not correspond to actual production metrics."
```

Comment:
This should be `**Disclaimer:** {...}` instead of a subheading.

Reply:
done